docs/index.rst

```diff
@@ -32,6 +32,16 @@ that allows students to check and instructors to grade assignments their own mac
 designed to grade Pyhon and R executables, Jupyter Notebooks, and RMarkdown documents and is 
 compatible with a few different LMSs, including Canvas and Gradescope.
 
+The core abstraction of Otter, as compared to other autograders like nbgrader_ and OkPy_, is this:
+you provide the compute, and Otter takes care of the rest. All a instructor needs to do in order to 
+autograde is find a place to run Otter (a server, a JupyterHub, their laptop, etc.) and Otter will
+take care of generating assignmetns and tests, creating and managing grading environents, and 
```

Comment:
s/assignmetns/assignments/

I'm not sure what's being implied here, as nbgrader isn't tied to JupyterHub, for example.

Reply:
Thanks for catching this! I'll try rewording this a bit.